docs/Using-the-dataset-validator.md

```diff
@@ -414,3 +414,78 @@ As an example, the command for the "Example 1" listed above incorporating the `-
 ```
 ./validateData.py -s ../../../test/scripts/test_data/study_es_0/ -P ../../../../../src/main/resources/portal.properties -u http://localhost:8080/cbioportal -v
 ```
+
+## Running the validator for multiple studies
+The importer folder `<your_cbioportal_dir>/core/src/main/scripts/importer` also contains a script for running the validator for multiple studies:
+```bash
+./validateStudies.py --help
+```
+The following parameters can be used:
+```console
+usage: validateStudies.py [-h] [-d ROOT_DIRECTORY] [-l LIST_OF_STUDIES]
+                          [-html HTML_FOLDER]
+                          [-u URL_SERVER | -p PORTAL_INFO_DIR | -n]
+                          [-P PORTAL_PROPERTIES]
+
+Wrapper where cBioPortal study validator is run for (multiple) input studies
+
+optional arguments:
+-h, --help            show this help message and exit
+-d ROOT_DIRECTORY, --root-directory ROOT_DIRECTORY
```

Comment:
Perhaps rename `root directory` to `parent directory`? I'm not sure. Root is usually the highest level in a hierarchy. What do you and @pieterlukasse think?